src/builder-context.h

```diff
@@ -22,14 +22,22 @@
 #define __BUILDER_CONTEXT_H__
 
 #include <gio/gio.h>
-#include <libsoup/soup.h>
 #include <curl/curl.h>
 #include "builder-options.h"
 #include "builder-utils.h"
 #include "builder-sdk-config.h"
 
 G_BEGIN_DECLS
 
+/* Same as SOUP_HTTP_URI_FLAGS, means all possible flags for http uris */
+
+#if GLIB_CHECK_VERSION (2, 68, 0) || !GLIB_CHECK_VERSION (2, 66, 0)
```

Comment:
```
#if GLIB_CHECK_VERSION (2, 68, 0)
```

Unlike Flatpak, flatpak-builder no longer supports versions older than 2.66 after this commit, and doesn't have a backport of GUri.